### DISCUSSION TITLE
Worldbuilding - Introduction and Preliminary Discussion

### DISCUSSION THREAD

**Discussion by Lilith-NightRose** [Thorium-Sim/thorium-nova#434] [General]
## Navigation
This discussion is divided into four parts:
> Introduction (you are here)
> [Governance Discussion](435)
> [Deliverables/Communication Discussion](https://github.com/Thorium-Sim/thorium-narrative/discussions/5)
> Caesium Universe Proposal (wip)

## What's going on here?
@alexanderson1993, @1deejay, and I (@Lilith-NightRose) had an excellent time Thursday (12/22) discussing worldbuilding for both Thorium Nova and Thorium Classic. The topics we discussed were wide-ranging, and not all topics were nailed down, but ultimately, we all share the goal of giving new users in Nova the opportunity to start flying _without_ having to develop a whole universe themselves. The next question is: how do we implement it?

## The Parts
This introduction contains an overview of what was discussed during the worldbuilding meeting on 12/22/2022, as well as placing several guiding markers for further discussions. The Governance discussion is on the topic of the best way to organize a collaborative worldbuilding process. Release & Communication addresses how we distribute information about the world we've built  with the release of Nova. On both these topics, I have some ideas, but not a solid recommendation. The fourth and final section is my specific proposal for a universe and the organization of a small "starter kit" and a more complete sourcebook for what I've codenamed the "Caesium Universe."

## Notes from the Worldbuilding Meeting
Alex, AdminAnonymous, and I discussed the problems and possibilities of the future of Thorium Nova on the nova-stage. You can find a recording of the meeting [here](https://drive.google.com/file/d/1oVULt-4vJUxxz_WxreGopMj2XKaujg4w/view).

Topics were wide-ranging, but two topics we kept coming back to were the tradeoffs between simplicity and nuance, and the interplay between mechanics and story. Both of these questions in part revolve the issue of playable and nonplayable "factions," and while we didn't come to any solid conclusions, the discussion helped narrow the field of questioning.

### Simplicity and Nuance
Simplicity is a core part of the [Vision](https://github.com/Thorium-Sim/thorium-nova/blob/develop/README.md) of Thorium Nova. In the [initial blog post](https://thoriumsim.com/blog/thorium-novas-lore) on worldbuilding and lore, Alex suggested that the best way to keep the lore simple might be to make make humans based on Earth/Terra a roughly unified "good guy" in the Galaxy, so that it's easy for new players to understand what they're fighting _for_ and what they're fighting _against_. However, some concerns were raised about the ways in which many classic RPGs and Fantasy worlds are struggling with the ways in which their "Evil" races/factions are often unintentionally associated with stereotypes of existing communities. Additionally, it was noted that, if not handled carefully, this basic paradigm can be difficult to break out of and add nuance to for more advanced players.

It was suggested that perhaps we want to start off with a universe where all factions are human. In addition to the benefits of avoiding xenophobia, this also allows a more natural-feeling interplay between mechanics and story (discussed further below). The conclusions we moved towards with these thorny questions are discussed in the 'factions' section of this discussion.

Another problem that was brought up with regard to the drive towards simplicity in Thorium lore development was the "Force Awakens" problem. This problem occurs when a simple introduction to a universe causes problems because nuance is not planned for. Here, I think we came to a loose consensus that the initial release of Thorium Nova Lore will come in two parts: an introductory mission pack, which comes with a relatively short and simple introduction to the relevant factions, and a larger "sourcebook," which allows Flight Directors and players to build off the initial knowledge gained in the introductory missions into more expansive take on the Thorium universe.

### Mechanics and Story
Thorium Nova is a work in progress, and we think the lore should be too. In D&D and Pathfinder, expansion packs build on the background information in early sources to add vibrancy to parts of the world that previously were mere footnotes.  This means the world can feel alive without worldbuilders having to plan out [Everything, Everywhere, All at Once](https://www.youtube.com/watch?v=wxN1T1uxQ2g).

There's another reason to want to allow the lore to build as we go: the dream is to allow players to play as different factions, and maybe even different species, but such experiences would be samey and bland unless there were accompanying mechanical/ship system changes to make it clear "oh, yeah, this vessel operates in an entirely different way." Expanding the lore over time allows Nova's initial release to contain one or two ship systems, but gives the opportunity to develop additional systems in coordination with lore and faction development!

### So, what's up with factions?
When starting out, what factions can crews play as? How do we communicate who are the "good guys" and the "bad guys"? Do such concepts even make sense? Ultimately, the questions of Mechanics and Story, Simplicity and Nuance all come down to trying to answer these and other questions about the different (and sometimes opposed) groups in the galaxy of Thorium

Ultimately, I think we started to move towards the conclusion that, while interacting with aliens can be a fun and exciting experience, initial missions should deal primarily with human factions, and that the preliminary lore release should mostly focus on these human factions. As players get deeper into the lore and the stories they build, they should feel empowered to choose to play more missions with whatever faction speaks to them.

As was discussed above, the hope is to have a "starting mission pack." What faction (and what _kind_ of faction) should these starting mission packs ally players with? Well, first of all, we want players to leave these missions (and others! but especially preliminary missions) feeling good about themselves and the choices they made. Bridge Simulations are, after all, inspired by Star Trek, the OG super-optimistic Sci-Fi show. We want players to feel like they can root for themselves. For that reason, early missions should generally be with factions who are easier to see as the "good guys," who are fighting for moral goals which most players will relate to.

### Crew variety
There's a second question though: what kind of crew will players be? Traditionally, the Space Centers have operated ships that are members of quasi-military fleets, with strict chains of command and rigorous protocols, but there are plenty of examples of futuristic space scenarios wherein there are small crews that are _not_ affiliated with a military or government (see, for example, Firefly). These independent crews share some similarities with the mercenary player parties of TTRPGs like D&D, Pathfinder, and Lancer.

So, which should it be? The conclusion we came to is that [both is good](https://media.tenor.com/ZezsFuiFB48AAAAd/both-is-good-both.gif). Two starter missions will be included, one that's set on an independent trading and/or mercenary ship with a relatively barebones crew, and one set on a naval ship with organizational backing. An additional benefit here is that each mission will have the opportunity to introduce different systems. The Firefly-style mission will introduce various automated systems necessary to make a ship run with just a bridge crew, while the Starfleet-style mission will introduce systems for crew management and other large-ship maintenance tasks.

## Conclusion and sneak peak
There is a consequence to the dual track approach discussed above:  whatever the lore ends up being, it needs to be a universe where both independent mercenary ships exist (and can make an impact) and in which major powers sometimes have major disagreements that can only be resolved by bigger-army-diplomacy. What should that look like? And how can a community-run project build an entire world? Well, we'd love if you give as many thoughts as you have below, and also...

Please join me over the next couple of days as I post a follow-up conversation about worldbuilding governance, and write up a brief introduction to my own Thorium lore proposal, which I've codenamed Caesium.

**Comment by Lilith-NightRose** [comment]
Oh, and one other thing that didn't really fit into the main body of the discussion: what is the connection between the lore for Nova and the Lore for Classic. Personally, I love classic, and would like to be able to share the world we're building with my friends well before Nova is feature-complete. For this reason, I'm hoping to make a version of the Lore sourcebook targeted at Classic.

**Reply by 1deejay** [reply]
@alexanderson1993 Did we want to create a label for the issues section for Narrative when we start to break down specific things to work on?

**Reply by alexanderson1993** [reply]
Gonna call it `Universe` so it encompasses all aspects of the lore, including narrative, gameplay, technology, and other aspects of the built-in content.

**Comment by 1deejay** [comment]
I think another thing to mention that got brought up is the fact that the built in universe is not intended to take over any other stories or universes that players want to make. In much the same way DnD and Pathfinder work, they have a more or less built in world, but someone decided to make Exandria and others make their own worlds. Players should feel empowered to make their own universes and worlds and stories. 

It's just in much the same way that creating a narrative greatly assisted the development of TTRPG systems, it seems just as much so creating a built in universe will help the development of Thorium.

**Comment by Science400** [comment]
As far as factions go, I like how they work in The Expanse. 

There are three main factions, Earth, Mars, and the Belt. Each of them is fighting to maintain their way of life. Each of them is fighting to exert dominance over the others. That makes it pretty easy to spin who is the good guy and who is the bad guy depending on how you approach the situation. 

In this worldbuilding project I can imagine each faction having nuanced motivations. That would make it easy to write that faction going either way. And then in that particular story their motivations become relatively straightforward and simple. If it ends up scaling to more than 3 factions, instead of everybody being everybody's ally and enemy at the same time, then some groups can be mostly allies, some mostly enemies, and some mostly neutral.

I also like how in the Expanse, the thing that defines what faction the character is a part of is having a set of shared experiences. Like growing up in 1g or eating a particular brand of nutrient kibble. Though, I'm not sure how to deal with that in a game like this where you're just dropped into the middle of everything.
